app/assets/stylesheets/styles/components/_download.scss

```diff
@@ -0,0 +1,37 @@
+.downloads {
+  &__downloads {
+    font-size: rem-calc(22);
+    font-weight: $semi-bold;
+
+    display: inline-block;
+  }
+
+  &__button {
+    max-width: rem-calc(264);
+
+    @include breakpoint($small) {
+      width: rem-calc(264);
+    }
+  }
+
+  &__li {
+    margin-bottom: rem-calc(30);
```

Comment:
Ok to leave this. The reason I commented I think was because it means the margin between the final download button and the about link below is formed from two different margins rather than in one place. Not a big thing though so we can leave.